src/main/api/ledger.ts

```diff
@@ -110,14 +111,55 @@ const signBtcTxInLedger = async (
   }
 }
 
+const signBNBTxInLedger = async (ledgerTxInfo: LedgerBNBTxInfo): Promise<E.Either<LedgerErrorId, string>> => {
+  const {
+    client,
+    sender,
+    asset,
+    amount,
+    recipient,
+    memo,
+    handleLedgerPresign,
+    handleLedgerVerifySuccess,
+    handleLedgerVerifyFailed
+  } = ledgerTxInfo
+  // await client.initChain()
+  const transport = await TransportNodeHid.open('')
+  const ledgerApp = new LedgerAppBNB(transport)
+  const derive_path = getBNBDerivePath(0)
+  client.useLedgerSigningDelegate(
+    ledgerApp,
+    () => {
+      if (handleLedgerPresign) handleLedgerPresign()
+    },
+    () => {
+      if (handleLedgerVerifySuccess) handleLedgerVerifySuccess()
+    },
+    (error) => {
+      if (handleLedgerVerifyFailed) handleLedgerVerifyFailed(getErrorId(error.message, error.statusText))
+    },
+    derive_path
+  )
+  const transferResult = await client.transfer(
+    sender,
+    recipient,
+    baseToAsset(amount).amount().toString(),
+    asset ? asset.symbol : AssetBNB.symbol,
+    memo
+  )
+  return E.right(transferResult.result.map((txResult: { hash?: TxHash }) => txResult?.hash ?? '')[0])
```

Comment:
[nitpick] `(txResult: { hash?: TxHash }) => hash | '')`

Just curious about `[0]`: Do we have more than just one result?